README.md

```diff
@@ -78,6 +79,13 @@ Note that these packages won't always have the exact names given above, as it ca
 
     # apt install cmake g++ libasound-dev libopenal-dev libdbus-1-dev libsdl2-dev libxkbcommon-dev waylandpp-dev
 
+### Arch Linux
+
+> [!NOTE]
+> [An AUR package is available](https://aur.archlinux.org/packages/surrealengine-git).
+
+    # pacman -S libx11 gcc git cmake sdl2 alsa-lib waylandpp
+
 Once you've installed all of the prerequisites, enter these commands in the given order from the folder you want to clone the repo to:
```

Comment:
I do think this part should be separated from the Arch Linux and Ubuntu instructions, since it's not specific to either of them. Maybe it would be good to title the Ubuntu and Arch Linux sections something like `Installing dependencies on Ubuntu/Arch Linux`. The below section could then have a title called `# Building` or something.

Reply:
Good point. I'll see what I can do :thinking:

Reply:
Updated. Is the new version better?

Reply:
Looks good